ccmbioinfo/lfs
# CARDS-2424 / CARDS-2427: Configuration of export exclusions

This needs to be merged with #1660

To test:
- build with `mvn clean install -Pdocker`
- in `compose-cluster` run `python3 generate_compose_yaml.py --dev_docker_image  --composum --cards_project cards4heracles --s3_test_container --oak_filesystem`
- start with `docker-compose build && docker-compose up`
- at `http://localhost:9001/` create a new bucket called `uhn`
- fill in a Phone call follow-up form, including details about the providers visited
- open `http://localhost:8080/Subjects.s3push?dateLowerBound=2020-01-01&dateUpperBound=2024-01-01` to trigger an export
- at `http://localhost:9001/` check the pushed file, make sure all the answer except for the providers are included in it
- stop docker and cleanup with `docker-compose rm -vf && ./cleanup.sh`
